templates/admin/home/index.html.twig

```diff
@@ -0,0 +1,197 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Tableau de bord - Admin{% endblock %}
+
+{% block body %}
```

Comment:
When the admin dashboard will be completed, we will have to move this menu in another file